wagtail_wordpress_import/block_builder_defaults.py

```diff
@@ -0,0 +1,290 @@
+            (
+                "h3",
+                {
+                    "FUNCTION": "wagtail_wordpress_import.block_builder_defaults.build_heading_block",
+                },
+            ),
+            (
+                "h4",
+                {
+                    "FUNCTION": "wagtail_wordpress_import.block_builder_defaults.build_heading_block",
+                },
+            ),
+            (
+                "h5",
+                {
+                    "FUNCTION": "wagtail_wordpress_import.block_builder_defaults.build_heading_block",
+                },
+            ),
+            (
+                "h6",
+                {
+                    "FUNCTION": "wagtail_wordpress_import.block_builder_defaults.build_heading_block",
+                },
+            ),
+            (
+                "table",
+                {
+                    "FUNCTION": "wagtail_wordpress_import.block_builder_defaults.build_table_block",
+                },
+            ),
+            (
+                "iframe",
+                {
+                    "FUNCTION": "wagtail_wordpress_import.block_builder_defaults.build_iframe_block",
+                },
+            ),
+            (
+                "form",
+                {
+                    "FUNCTION": "wagtail_wordpress_import.block_builder_defaults.build_form_block",
+                },
+            ),
+            (
+                "img",
+                {
+                    "FUNCTION": "wagtail_wordpress_import.block_builder_defaults.build_image_block",
+                },
+            ),
+            (
+                "blockquote",
+                {
+                    "FUNCTION": "wagtail_wordpress_import.block_builder_defaults.build_block_quote_block",
+                },
+            ),
+        ],
+    )
+
+
+"""Fall back StreamField block"""
+
+
+def conf_fallback_block():
+    return getattr(
+        settings,
+        "WAGTAIL_WORDPRESS_IMPORTER_FALLBACK_BLOCK",
+        "wagtail_wordpress_import.block_builder_defaults.build_none_block_content",
+    )
+
+
+def build_none_block_content(cache, blocks):
+    """
+    image_linker is called to link up and retrive the remote image
+    """
+    blocks.append({"type": "rich_text", "value": image_linker(cache)})
+    cache = ""
+    return cache
+
+
+"""Rich Text Functions"""
+
+
+def conf_valid_image_content_types():
+    return getattr(
+        settings,
+        "WAGTAIL_WORDPRESS_IMPORTER_VALID_IMAGE_CONTENT_TYPES",
+        [
+            "image/gif",
+            "image/jpeg",
+            "image/png",
+            "image/webp",
+            "text/html",
+        ],
+    )
+
+
+def conf_domain_prefix():
+
+    if hasattr(settings, "WAGTAIL_WORDPRESS_IMPORTER_BASE_URL"):
+        return getattr(settings, "WAGTAIL_WORDPRESS_IMPORTER_BASE_URL")
+
+    elif not hasattr(settings, "WAGTAIL_WORDPRESS_IMPORTER_BASE_URL") and hasattr(
+        settings, "BASE_URL"
+    ):
+        return getattr(settings, "BASE_URL")
```

Comment:
**Suggestion**: The `not` case is implicit if you use `elif`. We don't need to use `getattr` if we're inside an if block that has checked `hasattr`.

```suggestion
    if hasattr(settings, "WAGTAIL_WORDPRESS_IMPORTER_BASE_URL"):
        return settings.WAGTAIL_WORDPRESS_IMPORTER_BASE_URL
    elif hasattr(settings, "BASE_URL"):
        return settings.BASE_URL
```